cmd/xl-sets.go

```diff
@@ -1045,9 +1045,10 @@ func (s *xlSets) listObjectsNonSlash(ctx context.Context, bucket, prefix, marker
 	return result, nil
 }
 
-// ListObjects - implements listing of objects across disks, each disk is indepenently
+// ListObjects - implements listing of objects across disks, each disk is independently
 // walked and merged at this layer. Resulting value through the merge process sends
 // the data in lexically sorted order.
+// If heal is set only objects that does not have full quorum is returned.
```

Comment:
Could you rename 'heal' to 'avoidFullQuorum' here and in xlSets.listObjects() as well ?

Reply:
Good idea.

Reply:
@vadmeste Went for `partialQuorumOnly`.